<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotSettings</class>
 <widget class="QWidget" name="PlotSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Area</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbLandmarkName">
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QLineEdit" name="leYMin"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="leTitel"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>yMin:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="leXLabel"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leXMin"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>xLabel:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>xMin:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="leXMax"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>xMax:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>yMax:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Titel:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="leYMax"/>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QLineEdit" name="leYLabel"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>yLabel:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QPushButton" name="btChangePlottingparameter">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>cbLandmarkName</tabstop>
  <tabstop>leTitel</tabstop>
  <tabstop>leXMin</tabstop>
  <tabstop>leXMax</tabstop>
  <tabstop>leXLabel</tabstop>
  <tabstop>leYMin</tabstop>
  <tabstop>leYMax</tabstop>
  <tabstop>leYLabel</tabstop>
  <tabstop>btChangePlottingparameter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
